src/libvirt-xml-create.js

```diff
@@ -233,6 +236,39 @@ export function getSnapshotXML(name, description) {
         snapElem.appendChild(descriptionElem);
     }
 
+    if (isExternal) {
+        if (memoryPath) {
+            const memoryElem = doc.createElement('memory');
+            memoryElem.setAttribute('snapshot', 'external');
+            memoryElem.setAttribute('file', memoryPath);
+            snapElem.appendChild(memoryElem);
```

Comment:
These 6 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240126-102055-8ff71bb1-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)